### ISSUE TITLE
Skipped activations

### ISSUE THREAD

**Issue by AldusLorde** [Gofinge/CheXpert_Baseline_PyTorch#4]
Hi! Have you intentionally skipped activation functions in 'models.densenet' ?

**Comment by Gofinge** [comment]
Hi, I think for each kind of densenet, I used 'nn.Sigmoid()' as activation function for classification. (Line 18, 31, 44)

**Comment by AldusLorde** [comment]
Yeah, I see, but you used 2 linear units (15, 28, 41)
Example from your code:
   nn.Linear(num_features, num_features), #No activation after this unit
   nn.Dropout(p=0.9),
   nn.Linear(num_features, num_classes),
   nn.Sigmoid()

**Comment by Gofinge** [comment]
Okay, I got you. I was thought that the dropout layer can solve linear problem in some way, so I did not use an activation layer. But in my now opinion, there is no need to use two linear layers after DensenNet, since it doesn't make too much difference. I was too young too simple, haha.

You can try just one linear layer and one activation layer, I think it will also work and save a few time,

**Comment by AldusLorde** [comment]
Got it, thank you for your answers and time)

**Comment by Gofinge** [comment]
I have just analysised the code of torchvision.model.densenet121(), and found it use a linear layer for classification. You know my code add linear layer to change the num_classes, but it is not a good way, I think you can try change the num_class in torchvision.model.densenet121() through parameter 'num_classed'

**Comment by AldusLorde** [comment]
@Gofinge According to pre-implemented DenseNet, they skipped an activation function after the classifier module.
In my opinion, you made the right decision.
Though, my knowledge of PyTorch isn't good enough, so my thoughts can be wrong

**Comment by Gofinge** [comment]
Oh yes, meanwhile if I change it, I can't use the pre-trained model.

**Comment by Gofinge** [comment]
You can try it. Just add a activition layer after raw model and don't use the pretrained parameters.

**Comment by AldusLorde** [comment]
Ok, I'll try it and write you about results
Thank you for your help)

**Comment by Gofinge** [comment]
Looking forward to your feedback!
